docs/guide/modules.md

```diff
@@ -0,0 +1,21 @@
+# Modules
+
+::: warning TODOs
+:warning: This section is in a work in progress :warning:
```

Comment:
```suggestion
:warning: This section is a work in progress :warning:
```

Reply:
(can't you componentize this ?)

Reply:
Sure! But don't want to let them in place for a long period.

Best option is to finish these sections.

Reply:
We could need it in the future though